echopop/survey.py

```diff
@@ -1250,7 +1252,7 @@ def krige( self ,
         ### TODO: This should be refactored out as an external function 
         ###### rather than a Survey-class method.
         ### Apportion biomass based on age and sex
-        self.apportion_kriged_biomass( species_id )
+        # self.apportion_kriged_biomass( species_id )
```

Comment:
I guess this will be something that gets resolved once #215 is merged?

Reply:
Yep! The plan is to uncomment this once the kriging rework is completed re: [#202](https://github.com/OSOceanAcoustics/echopop/issues/202).